src/nemo_safe_synthesizer/evaluation/assets/css/multi_modal_report.css

```
@@ -1,564 +1,1078 @@
 }
 
-div.field {
-    display: inline;
-    font-weight: 700;
-    size: 16px;
-    line-height: 32px;
+.dataset-tag {
+    display: inline-flex;
+    margin-bottom: 14px;
+    padding: 2px 12px;
+    border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)